lib/theme/typography.ts

```diff
@@ -2,6 +2,7 @@ import { css } from "@emotion/css";
 import breakpoints from "./breakpoints";
 
 export const h1 = css`
+  font-family: "Poppins";
```

Comment:
What do you think of setting `font-family` as a css var?

And probably a question for @Go-Mori, is it worth having a fallback font?